### PR TITLE
Solution for `Add Todo Form`

### PR DESCRIPTION
**Pull request by jstmpelowycz**
**[DEMO LINK](https://jstmpelowycz.github.io/react_add-todo-form/)**

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import '../../styles/components/TodoInfo.scss';
+import { FC, memo } from 'react';
+import { Form } from 'react-bootstrap';
+import FormCheckLabel from 'react-bootstrap/FormCheckLabel';
```
Comment (mariiishka) [inline src/components/TodoInfo/TodoInfo.tsx:4]: 🔥

File: src/App.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import "styles/style";
+
+.page{
```
Comment (mariiishka) [inline src/App.scss:3]: filename must match the block name

**Review by mariiishka — APPROVED** [review]
awesome UI !

File: src/App.scss (hunk 2 above)
Comment (jstmpelowycz) [reply src/App.scss:3]: Yap, my bad, thank you!

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Css file should place together with component

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**